scripts/us_fda/drugs/create_enums.py

```diff
@@ -0,0 +1,453 @@
+        'AT2':
+            'Considered to be therapeutically equivalent to other drugs also coded with AT2.',
+        'BC':
+            'Extended-release dosage forms (capsules, injectables and tablets). Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BD':
+            'Active ingredients and dosage forms with documented bioequivalence problems. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BE':
+            'Delayed‑release oral dosage forms. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BN':
+            'Products in aerosol-nebulizer drug delivery systems. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BP':
+            'Active ingredients and dosage forms with potential bioequivalence problems. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BR':
+            'Suppositories or enemas that deliver drugs for systemic absorption. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BS':
+            'Products having drug standard deficiencies. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BT':
+            'Topical products with bioequivalence issues. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.',
+        'BX':
+            'Drug products for which the data are insufficient to determine therapeutic equivalence. Considered not to be therapeutically equivalent to other pharmaceutically equivalent products.'
+    }
+
+    for te_code in te_code_to_description:
+        # append enum to dcid relationship to dcid_dict
+        dcid_dict[te_code] = 'TherapeuticEquivalenceCode' + te_code
+
+        #write each enum to file
+        if enum_schema_file:
+            te_code_mcf = '\nNode: dcid:TherapeuticEquivalenceCode' + te_code + '\n'
+            te_code_mcf = te_code_mcf + 'typeOf: TherapeuticEquivalenceCodeEnum\n'
+            te_code_mcf = te_code_mcf + 'name: “TherapeuticEquivalenceCode' + te_code + '”\n'
+            te_code_mcf = te_code_mcf + 'description: “' + te_code_to_description[
+                te_code] + '”\n'
+            te_code_mcf = te_code_mcf + 'descriptionURL: “https://www.fda.gov/drugs/development-approval-process-drugs/orange-book-preface#TEC”\n'
+
+            if not te_code.isalpha():
+                te_code_mcf = te_code_mcf + 'specializationOf: dcs:TherapeuticEquivalenceCode' + te_code[
+                    0:2] + '\n'
+
+            enum_schema_file.write(te_code_mcf)
+
+
+# beginning of the final enum schema mcf file, contains Class definition for
+# each enum type, marketing status enums, and application type enums
+ENUMS_HANDWRITTEN = '''
```

Comment:
Can we place these large constant blocks and maps in a separate create_enum_constants.py ? That way, the code portions are shorter and easier to follow.